shared/chat/blocking/invitation-to-block.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import {WaveButton} from '../../settings/contacts-joined/buttons'
+import * as Styles from '../../styles'
+type Props = {team?: string; usernames?: Array<string>}
+
+const BlockButtons = (props: Props) => {
+  return (
+    <Kb.Box2 direction="vertical" gap="tiny" centerChildren={true}>
+      <Kb.Text type="BodySmall">
+        {props.team ? 'Someone added you to this team.' : "You don't seem to know the sender."}
```

Comment:
There was a buried note in zeplin that we should show the username who added you. Seems it'd correspond to the `adder` prop on the modal